File: App/Backend/Backend/Backend/Models/PostLike.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Backend.Models
{
    public class PostLike
    {
        [Key]
        public int Id { get; set; }
        public int PostId { get; set; }
        public String UserName { get; set; }
        public DateTime Time { get; set; }
    }
}

File: App/Frontend/frontendWeb/lib/models/postLike.dart
class PostLike{
  int _postId;
  String _userName;
  DateTime _time;

  PostLike(this._postId, this._userName){
    this._time = DateTime.now();
  }

  Map<String, dynamic> toJSON(){
    var map = Map<String, dynamic>();

    map["postId"] = _postId;
    map["userName"] = _userName;
    map["time"] = _time.toIso8601String();

    return map;
  }
}